src/Faker/ORM/Doctrine/Populator.php

```diff
@@ -98,11 +98,9 @@ public function execute($entityManager = null)
                 );
                 if (count($insertedEntities) % $this->batchSize === 0) {
                     $entityManager->flush();
-                    $entityManager->clear($class);
```

Comment:
This is totally unrelated?

Reply:
This is from already merged commit (https://github.com/fzaninotto/Faker/commit/5337ce5261b393937bd7aaebb21020159756a152), just because I branched the repo before.